### ISSUE TITLE
Sushi Go Crafting

### ISSUE THREAD

**Issue by Buuz135** [EnigmaticaModpacks/Enigmatica10#87]
### URL

https://www.curseforge.com/minecraft/mc-mods/sushigocrafting

### Why would you like the mod added?

So boo cant stop asking for stuff

**Comment by MuteTiefling** [comment]
Cat?

**Comment by Buuz135** [comment]
No!

**Comment by MuteTiefling** [comment]
😭

**Comment by EAGBCD** [comment]
I don't believe this was ever added despite being closed and 'fixed in next release'. I would like to +1 to it being included in a future update.

**Comment by NielsPilgaard** [comment]
Woops, let's try again!
